react-charts/area-chart/intro.md

```diff
@@ -1 +1,15 @@
-In this scenario, you will learn how to use PatternFly React area chart components for the first time.
+In this scenario, you will learn how to use PatternFly React area chart components.
+
+You will begin by adding a CSS selector to define the chart size...
+
+<strong>Note:</strong> Katacoda is setting up a new React application for you. You will be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
+
+
+Because Victory renders responsive containers, the Chart's width and height props do not determine the width and height of the chart in number of pixels, but instead define an aspect ratio for the chart. 
+
+- The exact number of pixels will depend on the size of the container the chart is rendered into. 
+
+- Typically, the parent container is set to the same width in order to maintain the aspect ratio.
```

Comment:
Let's change "in order to" to just "to." Whenever we can get away with shortening "in order to" to "to," let's run with it! We use less words and sound more casual.